### PR TITLE
Refactor export GPG_TTY command to add line break (#25721)

### LINKED ISSUE #25721: Shell command to add env var breaks if file ends in a comment
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

Step 7 on [Tell Git about your signing key](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key) requires running this:

```
$ if [ -r ~/.zshrc ]; then echo 'export GPG_TTY=$(tty)' >> ~/.zshrc; \
  else echo 'export GPG_TTY=$(tty)' >> ~/.zprofile; fi
```

This does not work if the user's `.zshrc` or `.zprofile` file ends in a comment. I recently installed pnpm so my `.zshrc` files ends with:
```
# pnpm end
```

### What part(s) of the article would you like to see updated?

Update the commands in step 7 for Mac and Linux to add a linebreak before the export. 
```
echo -e '\nexport GPG_TTY=\$(tty)'
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by AlmostEfficient**
The command to add `GPG_TTY` does not update the shell config correctly if the file ends in a comment.
```
$ if [ -r ~/.zshrc ]; then echo 'export GPG_TTY=$(tty)' >> ~/.zshrc; \
  else echo 'export GPG_TTY=$(tty)' >> ~/.zprofile; fi
```

I previously installed pnpm so running this changed my  `.zshrc` file to:
```
#pnpm endexport GPG_TTY=$(tty)
```

### Why:

Closes: #25721 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Fixed by updating each instance of the command **from**:

```
echo 'export GPG_TTY=$(tty)'
```

**to:**

```
echo -e '\nexport GPG_TTY=\$(tty)'
```

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@AlmostEfficient Thank you for opening a PR and linking it to your issue! ✨ 

We appreciate the extra context, too. I'll get this triaged for review! :zap:

**Review by skedwards88 — DISMISSED** [review]
Thanks for this improvement! We'll get this merged down for you.

File: content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md [hunk 1]
```diff
@@ -96,6 +96,9 @@ If you have multiple GPG keys, you need to tell Git which one to use.
 {% data reusables.gpg.paste-gpg-key-id %}
 {% data reusables.gpg.set-auto-sign %}
 1. To add your GPG key to your `.bashrc` startup file, run the following command:
+   ```bash
+   $ [ -f ~/.bashrc ] && echo -e '\nexport GPG_TTY=\$(tty)' >> ~/.bashrc
+   ```
    ```bash
    $ [ -f ~/.bashrc ] && echo 'export GPG_TTY=$(tty)' >> ~/.bashrc
    ```
```
Comment (skedwards88) [inline content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md:104]: My bad misresolving the merge conflict here
```suggestion
```

**Review by skedwards88 — APPROVED** [review]

**Review by Brandonlb91 — COMMENTED** [review]
# content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md
-git-about-your-signing-key.md

**Pull request merged**